### ISSUE TITLE
Azure.Template.UseDescriptions is reporting an error which we cannot determine the cause of

### ISSUE THREAD

**Issue by ianbrook** [Azure/PSRule.Rules.Azure#1263]
Files attached

Pipeline Output
##[error]ArtefactDownload/apptools.json failed Azure.Template.UseDescriptions. Use descriptions for each resource in generated template(bicep, psarm, AzOps) to communicate purpose

[apptools.zip](https://github.com/Azure/PSRule.Rules.Azure/files/8083529/apptools.zip)

**Comment by BernieWhite** [comment]
@ianbrook Thanks for logging this.

It's to do with resources in your modules.

For example this route table.

```json
{
  "type": "Microsoft.Network/routeTables",
  "apiVersion": "2020-07-01",
  "name": "[parameters('routeTableName')]",
  "location": "[parameters('location')]",
  "tags": "[parameters('tags')]",
  "properties": {
    "routes": "[parameters('routes')]",
    "disableBgpRoutePropagation": "[parameters('disableBgpPropagation')]"
  }
}
```

In the Bicep code of the module use the `@description('A description')` on each resource so someone reading the code understand the purpose.

Please keep this one open and we'll make sure we update the guidance to make this clearer.
